onnx/checker.h

```diff
@@ -81,6 +89,7 @@ class CheckerContext final {
   bool is_main_graph_ = true;
   const ISchemaRegistry* schema_registry_ = OpSchemaRegistry::Instance();
   std::string model_dir_;
+  bool skip_opset_compatibility_check_ = false;
```

Comment:
Why use the prefix skip?

Reply:
I find it less ambiguous (than something like `opset_compatibility_check`).